src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -242,17 +250,39 @@ void convertRequireToImportStatements(Node n, String fullLocalName, String requi
 
     boolean imported = false;
     if (module.importedNamespacesToSymbols.containsKey(requiredNamespace)) {
-      // import {value as localName} from "./file"
-      Node importSpec = new Node(Token.IMPORT_SPEC, IR.name(moduleSuffix));
-      // import {a as b} only when a =/= b
-      if (!moduleSuffix.equals(localName)) {
-        importSpec.addChildToBack(IR.name(localName));
+      @Nullable String defaultExportSymbol = module.exportedNamespacesToSymbols.get(EXPORTS);
+      String importedSymbol = module.importedNamespacesToSymbols.get(requiredNamespace);
+      boolean isDefaultExport = importedSymbol.equals(defaultExportSymbol);
+
+      Optional<Node> defaultExport;
```

Comment:
I actually see that as an anti-pattern, since static analyzers are unable to determine if it was set in each branch.